python/tvm/relay/frontend/tflite.py

```diff
@@ -1067,6 +1069,27 @@ def convert_zeros_like(self, op):
 
         return out
 
+    def convert_fill(self, op):
+        """Convert TFLite FILL"""
+        try:
+            from tflite.Operator import Operator
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
```

Comment:
Please drop this , it's no longer needed as the assert has now been commonised.

Reply:
Merged master and removed.